registration.js

```diff
@@ -0,0 +1,66 @@
+function validate() {
+
+    let objForm = {
+        SignupName,
+        SignupSurname,
+        SignupBirthday,
+        SignupEmail,
+        SignupPassword,
+        SignupConfirmPassword
+    };
+
+    for (let key in objForm) {
+        const node = document.getElementById(key).value;
+
+        if (node) {
+            objForm[key] = node;
+        }
+    }
+
+    function warning(x) {
+        let warningNames = {
+            name: `Некорректный ввод!`,
+            birthday: `10лет < Ваш возраст > 80лет`,
+            email: `Ошибка ввода Email-адреса!`,
+            wrongSimbols: `Invalid Password!`,
+            password: `Пароли не совпадают`
+        };
+        document.getElementById(`warning`).textContent = warningNames[x];
+    }
+
+    function isValidDate(year) {
+        year = new Date().getFullYear() - new Date(year.replace(/\./g, `:`)).getFullYear();
+        return (10 < year && year < 80) ? true : false;
+    }
+
+    let regEmail = /[0-9a-zа-я_A-ZА-Я]+@[0-9a-zа-я_A-ZА-Я^.]+\.[a-zа-яА-ЯA-Z]{2,4}/i,
+        regName = /^([а-яА-ЯA-Za-z]{2,20})+$/,
+        regPassword = /^([а-яА-ЯA-Za-z0-9]{6,})+$/;
+
+
+    if (regName.test(objForm.SignupName) == false){
+        warning(`name`);
+        return false;
+    }
+    if (regName.test(objForm.SignupSurname) == false){
+        warning(`name`);
+        return false;
+    }
+    if (isValidDate(objForm.SignupBirthday) == false){
+        warning(`birthday`);
+        return false;
+    }
+    if (regEmail.test(objForm.SignupEmail) == false){
+        warning(`email`);
+        return false;
+    }
+    if (regPassword.test(objForm.SignupPassword) == false) {
+        warning(`wrongSimbols`);
+        return false;
+    }
+    if(objForm.SignupPassword !== objForm.SignupConfirmPassword){
+        warning(`password`);
+        return false;
+    }
```

Comment:
Rewrite it in loop-like code.
Use shorthand boolean methods, like `if (!regName.test(objForm.SignupName))`.